README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Title
 
-## Available Scripts
+Math Magicians
 
-In the project directory, you can run:
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations.This is the first project of the Math Magicians application, implementation of set up the environment and tools needed to develop a React application. 
 
-### `npm start`
+<!-- ![screenshot](./src/images/app_screenshot1.png)
+![screenshot](./src/images/app_screenshot2.png) -->
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Built With
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- React
 
-### `npm test`
+<!-- ## Live Demo
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Live Demo Link](https://gegardus.github.io/JS-Capstone-Webapp/) -->
```

Comment:
- [ ] OPTIONAL: I highly suggest your remove these comments from the readme file.  That way, the project will be cleaner 👍